documentation/staging/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -227,6 +230,8 @@ the operator will not override the environment variable based on the shutdown co
 | `SHUTDOWN_TYPE` | `Graceful` | `Graceful` or `Forced` |
 | `SHUTDOWN_TIMEOUT` | 30 | Whole number in seconds where 0 means no timeout |
 | `SHUTDOWN_IGNORE_SESSIONS` | `false` | Boolean indicating if active sessions should be ignored; only applicable if shutdown is graceful |
+| `SHUTDOWN_WAIT_FOR_ALL_SESSIONS` | `false` | `true` to wait for all HTTP sessions during inflight work handling; `false` to wait for non-persisted HTTP sessions only ; only applicable if shutdown is graceful |
+
```

Comment:
Why is there a serverPod.waitForAllSessions domain resource configurable when all of the other shutdown configurable's use env vars?  For example, it is confusing to say `For graceful shutdown only` in theserverPod.waitForAllSessions resources without having a sister configuration at same scope for configuring shutdown. IMO, they either all should have a `serverPod` setting or none should...

Reply:
Ignore this. I didn't notice that there are already sister settings.